features/contents-attr.yml

```
@@ -0,0 +1,6 @@
+name: "content: attr()"
```

I'm suggesting this so that this feature sorts together with its broader counterpart:

```suggestion
name: attr() (content only)
```